python/pyspark/pandas/tests/data_type_ops/test_binary_ops.py

```diff
@@ -147,6 +147,14 @@ def test_from_to_pandas(self):
         self.assert_eq(pser, psser.to_pandas())
         self.assert_eq(ps.from_pandas(pser), psser)
 
+    def test_astype(self):
+        pser = self.pser
+        psser = self.psser
+        self.assert_eq(pd.Series(["1", "2", "3"]), psser.astype(str))
+        self.assert_eq(pser.astype("category"), psser.astype("category"))
+        cat_type = pd.api.types.CategoricalDtype(categories=[b"2", b"3", b"1"])
```

Comment:
nit: Shall we import `CategoricalDtype` at the header? And ditto in the other files.

Reply:
Certainly.